src/main/java/dev/openfeature/sdk/MutableContext.java

```diff
@@ -0,0 +1,142 @@
+package dev.openfeature.sdk;
+
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import lombok.experimental.Delegate;
+
+@ToString
+@SuppressWarnings("PMD.BeanMembersShouldSerialize")
+public class MutableContext implements EvaluationContext {
```

Comment:
Can I get a java doc to understand what this really is and how it differs from EvaluationContext?